app/controllers/repository_stock_values_controller.rb

```diff
@@ -45,7 +47,10 @@ def create_or_update
       )
     end
 
-    render json: @repository_stock_value
+    render json: {
```

Comment:
Why just not use serializer?

Reply:
Because it needs to be compatible with the DataTable cell to render it properly. These things are not in a serializer, but in the repository datatable service.